Integrations/MACVendors/MACVendors - Test.yml

```diff
@@ -0,0 +1,229 @@
+id: MACVendors - Test
```

Comment:
this file should be in `TestPlaybooks` directory and should have `playbook-` prefix